docs/website/docs/general-usage/naming-convention.md

```diff
@@ -141,6 +141,24 @@ naming="tests.common.cases.normalizers.sql_upper"
 ```
 `dlt` will import `tests.common.cases.normalizers.sql_upper` and use the `NamingConvention` class found in it as the naming convention.
 
+:::tip
+Do not pass custom naming convention as modules if you do it explicitly. We recommend pattern below:
+```py
+import dlt
+
+# you should be able to import sql_cs_latin2 here!
```

Comment:
Maybe remove this line? It's confusing, you say do not import it as modules, and then you say `you should be able to import sql_cs_latin2 here!`